### PR TITLE
Bug/abd storyedit route

### LINKED ISSUE #72: Fix bug that breaks story edit mode

### PR DESCRIPTION
**Pull request by Abdeboskey**
#### What's this PR do?
* Fix bug that prevents a logged-in user from accessing the story edit view.
* Also fix bug where App was not re-rendering the contents of newly published stories. 
#### Where should the reviewer start?
* Begin in App.js
#### How should this be manually tested?
* Launch the application and test out the user flow: login, start a new story or continue a story, and then make sure that the new story shows up with the correct details on the welcomepage
#### Any background context you want to provide?
* stumbled on this bug while beginning to style the story edit view
#### What are the relevant tickets?
Closes #72

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react';
+import React, { Component, useEffect } from 'react';
```
Comment (carlymclift) [inline src/components/App/App.js:1]: What does useEffect do?

**Review by carlymclift — APPROVED** [review]
Looks great! thanks for this!

File: src/components/App/App.js (hunk 1 above)
Comment (nickhartdev) [reply src/components/App/App.js:1]: Ah we forgot to take that out! At one point we were trying to use a hook to problem solve

**Review by nickhartdev — APPROVED** [review]
Was here for this! Thanks for driving Aaron!

**Pull request merged**